.github/workflows/coverage.yml

```
@@ -52,7 +69,11 @@ jobs:
             REPO_NAME=$(basename "$REPO")
 
             echo "→ Cloning $REPO..."
-            git clone --depth 1 "https://github.com/$REPO.git"
+            if ! git clone --depth 1 "https://github.com/$REPO.git"; then
+              echo "    ❌ Failed to clone $REPO, skipping..."
+              echo "  {\"repo\": \"$REPO\", \"coverage\": null, \"status\": \"clone_failed\" }," >> ../../_temp_coverage.json
+              continue
+            fi
```

I think we should revert this and let the CI fail in the event our config contains a typo on a repo, for example.

---

removed